backend/src/main/java/com/social/dailylink/controllers/AuthController.java

```diff
@@ -94,28 +95,26 @@ public ResponseEntity<?> registerUser(@Valid @RequestBody SignupRequest signUpRe
     Set<String> strRoles = signUpRequest.getRole();
     Set<Role> roles = new HashSet<>();
 
-    final String errorRoleString = "Error: Role is not found.";
-
     if (strRoles == null) {
       Role userRole = roleRepository.findByName(ERole.ROLE_USER)
-          .orElseThrow(() -> new RuntimeException(errorRoleString));
+          .orElseThrow(() -> new RuntimeException(Global.GLOBAL_STRINGS.getERROR_ROLE_NOT_FOUND()));
```

Comment:
Strange getters! Are they needed? Can't we use static variables if they are final?

Reply:
+1